vars/dockerBuildArgs.groovy

```diff
@@ -40,7 +44,9 @@ String call(Map config = [:]) {
               " --build-arg UID=" + getuid() +
               " --build-arg JENKINS_URL=$env.JENKINS_URL"
     if (cachebust) {
+      Calendar current_time = Calendar.getInstance()
       ret_str += " --build-arg CACHEBUST=${currentBuild.startTimeInMillis}"
+      ret_str += " --build-arg CB0=" + current_time.get(Calendar.WEEK_OF_YEAR)
```

Comment:
Why not be more transparent about it?
```suggestion
      ret_str += " --build-arg CB_WEEKLY=" + current_time.get(Calendar.WEEK_OF_YEAR)
```

Then, when reading the `Dockerfile`, it will be clear when to expect that to trigger.

Reply:
Because we might change our policy on how often we do this, I've been deliberately not littering references to it throughout the code.   The checkpatch dockerfile is already using CB0 so that would have to be changed but we could do that if you felt it was worth it.  One of the advantages if having this in pipeline-lib actually is it would be easier to do this.

Reply:
Done, and I've pushed a new version to daos as well.

Reply:
> Because we might change our policy on how often we do this,

Fair point.  Or other users of the Dockerfile can apply their own value to `CB0` to implement their own policy of how frequently to update at the `CB0` interval.

Reply:
> Done, and I've pushed a new version to daos as well.

Apologies for only getting to this PR now.  Just had to many plates spinning the last day or two.  I actually don't disagree with your reasoning for keeping it more generic, such as `CB0` is.  So if you wanted to unwind the most recent change, I wouldn't have a problem with that.  I wonder if we can come up with something more descriptive as to what `CB0` is actually meant to be though, so that it's more obvious from the outside, rather than having to look through Dockerfiles to figure it out.  Maybe there is no better name.

Reply:
On reflection I'll change this back to CB0, as you say we intend for other users to be able to use these and a documented CB0 meets that need better.  I'll add a one-liner comment to the Dockerfiles next to it.

# Option to periodically force refresh of entire image, to avoid later updates growing unbounded.